usagov/usagov-2021
# USAGOV-1880: benefits redirects

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1180

## Description

Nginx configuration updates. Includes use of "map" instead of "rewrite" for a large number of redirects. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions




### Change Requirements



### Validation Steps

I've done some smoke testing in the dev environment. Optionally we could merge this into the "dr" branch and deploy it there. 

Do lay eyes on the whole thing to see that it's organized in a way that makes sense (at least as much as the previous version did).

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
